### PR TITLE
Fix bad and/or broken links in the documentation

### PR DESCRIPTION
**Pull request by prince-chrismc**
## Description

Fixes pyrsia/pyrsia#713

I reviewed all the bad link and put ones that made more sense

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).
- [x] I've requested a review  from `pyrsia/collaborators`.

### CODE REVIEW

**Review by fishseabowl — DISMISSED** [review]
LGTM

File: docs/get_involved/rfc/readme.md [hunk 1]
```diff
@@ -76,7 +76,8 @@ Build consensus and integrate feedback. Proposals that have broad support are mu
 ### Reviewing a Proposal
 
 Periodically, the team will attempt to review the active proposals. We try to dicuss proposals at the bi-weekly team
-["Architecture"](https://pyrsia.io/events/) meeting, we schedule additional meetings as need. Actions are recorded in the meeting minutes.
+["Architecture"](https://pyrsia.io/docs/get_involved/#attend-a-community-meeting) meeting, we schedule additional meetings as need. Actions are recorded in the meeting minutes
```
Comment (erwin1) [inline docs/get_involved/rfc/readme.md:79]:
```suggestion
["Architecture"](https://pyrsia.io/docs/get_involved/#attend-a-community-meeting) meeting, we schedule additional meetings as needed. Actions are recorded in the meeting minutes
```
?

File: docs/readme.md [hunk 2]
```diff
@@ -4,5 +4,11 @@ sidebar_position: 0
 
 # Documentation Overview
 
-- [Tutorials](/docs/tutorials)
-- [Get Involved](/docs/get_involved)
+Welcome to Pyrsia's documentation, here you can explore how to install and run your very own Node. You can also
```
Comment (erwin1) [inline docs/readme.md:7]: Pyrsia Node instead of Node? (just to void confusion with that other Node)

**Review by erwin1 — COMMENTED** [review]
just a few minor comment, but looks good for the rest

**Review by erwin1 — APPROVED** [review]

**Comment by prince-chrismc** [comment]
Links are passing again 🎉

**Review by fishseabowl — APPROVED** [review]
LGTM

**Review by fishseabowl — DISMISSED** [review]

File: docs/tutorials/setup_on_macosx.md [hunk 3]
```diff
@@ -237,5 +237,5 @@ Success!!!
 You have just built yourself a working Pyrsia network. Enjoy using it and
 showcasing it to your teams and please share any feedback!
 
-Next you can, follow the instructions in [demo.md](https://pyrsia.io/tutorials/demo/)
+Next you can, follow the instructions in git push --set-upstream origin set-npm-ver-req
```
[the remark this answers is not in the document]
Comment (erwin1) [reply docs/tutorials/setup_on_macosx.md:240]:
```suggestion
Next you can follow the [demo instructions](https://pyrsia.io/docs/tutorials/demo/)
```

[the remark this answers is not in the document]
Comment (prince-chrismc) [reply docs/tutorials/setup_on_macosx.md]: Whoops must have type in the wrong window

**Review by erwin1 — APPROVED** [review]

**Pull request merged**